docs/monitors/host-metadata.md

```diff
@@ -0,0 +1,53 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# host-metadata
+
+ This monitor collects metadata about a host.  It is
+required for some views in SignalFx to operate.
+
+```yaml
+monitors:
+  - type: host-metadata
+```
+
+In containerized environments host `/etc` and `/proc` may not be located
+directly under the root path.  You can specify the path to `proc` and `etc`
+using the monitor configurations `procFSPath` and `etcPath`
+
+```yaml
+monitors:
+  - type: host-metadata
+    procFSPath: "/hostfs/proc"
+    etcPath: "/hostfs/etc"
+```
+
+
+Monitor Type: `host-metadata`
+
+[Monitor Source Code](https://github.com/signalfx/signalfx-agent/tree/master/internal/monitors/metadata/hostmetadata)
+
+**Accepts Endpoints**: No
+
+**Multiple Instances Allowed**: Yes
+
+## Configuration
+
+| Config option | Required | Type | Description |
+| --- | --- | --- | --- |
+| `procFSPath` | no | `string` | The path to the proc filesystem. Useful to override in containerized environments. (**default:** `/proc`) |
+| `etcPath` | no | `string` | The path to the main host config dir. Useful to override in containerized environments. (**default:** `/etc`) |
+
+
+
+
+## Metrics
+
+This monitor emits the following metrics.  Note that configuration options may
+cause only a subset of metrics to be emitted.
+
+| Name | Type | Description |
+| ---  | ---  | ---         |
+| `gauge.sf.host-plugin_uptime` | gauge | The time this monitor has been running in milliseconds.  Dimensions include `signalfx_agent`, `collectd`, `kernel_release`, `kernel_version` |
```

Comment:
Dimensions should be documented using the special Dimension note comments where you can give a more complete description of them.